src/SFA.DAS.AssessorService.Web/Controllers/ApplyForWithdrawalController.cs

```diff
@@ -114,10 +119,17 @@ public async Task<IActionResult> ChooseStandardForWithdrawal()
                 ? result
                 : (int?)null;
 
+            // remove any standards where there is an existing withdrawal application
```

Comment:
You should not remove from the list any standards, whether or not they have an in progress withdrawal / withdrawal by version. Instead all should be listed and if the user selects one that has an in progress withdraw they should be informed with a Warning message. This maay require input from the UX team.

Reply:
This is no longer applicable with the UI changes